dev/automated_tests/flutter_test/print_user_created_ancestor_expectation.txt

```diff
@@ -1,8 +1,6 @@
 <<skip until matching line>>
 ══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
-The following assertion was thrown building
-RawGestureDetector-\[LabeledGlobalKey<RawGestureDetectorState>#.+\]\(state:
-RawGestureDetectorState#.+\(gestures: <none>, behavior: opaque\)\):
+<<skip until matching line>>
```

Comment:
lgtm

Reply:
why this change?

Reply:
Because the refactor out, the error is throw at a lower level. As the goal of this test, it only want to make sure the widget inspector part is correct. That is why I am ok with this.

Reply:
@Hixie I also included an explanation in PR description _Regarding test change_

Reply:
tests don't just check what the original author thought they were testing. They test that things work. I think in this case making sure that the message continues to make sense is a good thing to test. Otherwise we're just reducing our test coverage.

Reply:
I found out that this break is due to the fact that `Listener` currently nests `_PointerListener`, its actual logic, with `MouseRegion`, the deprecated part of behavior. If we rename `_PointerListener` to `Listener`, then the behavior resumes, which will be applied when we remove the deprecation in the future. I think it shows that this part of expectation tests is an over-test and goes against the philosophy of composition.

Reply:
There's no such thing as an "overtest". :-)

I agree that we will want to change this again in the future, but the point is that we _don't_ want to change this _accidentally_ at some _other_ time, and having this test would catch us doing that.

Reply:
Ok, I've changed the lines to `The following assertion was thrown building Listener`